smoke-tests/src/test/groovy/io/opentelemetry/smoketest/SmokeTest.groovy

```diff
@@ -102,6 +102,7 @@ abstract class SmokeTest extends Specification {
       .withEnv("OTEL_BSP_MAX_EXPORT_BATCH_SIZE", "1")
       .withEnv("OTEL_BSP_SCHEDULE_DELAY_MILLIS", "10")
       .withEnv("OTEL_EXPORTER_OTLP_ENDPOINT", "collector:55680")
+      .withEnv("OTEL_EXPORTER_OTLP_INSECURE", "true")
```

Comment:
curious why this is now required. Are we defaulting to requiring TLS somewhere?

Reply:
Oh shoot, I had meant to flag you on this and forgot, nice eyes. 

Yeah because of my use of a primitive type instead of nullable, it ended up defaulting to TLS with autoconfigure

https://github.com/open-telemetry/opentelemetry-java/blob/v0.14.1/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java#L55

Luckily with 0.15 it'll only use the scheme from the endpoint

https://github.com/open-telemetry/opentelemetry-java/blob/master/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java#L50

Reply:
@breedx-splk and I ran into this today while setting doing some benchmarking work around some possible tweaks to the jdbc instrumentation. Definitely confused us for a while!

Reply:
@jkwatson Oh - yeah I guess it affects HEAD agent right now, but won't affect the released 0.15.0 since we'll migrate to the scheme by then.